specification/repository.md

```diff
@@ -77,20 +77,25 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
 
 - MUST have a [CHANGELOG.md](templates/CHANGELOG.md) updated for every release
   - The `CHANGELOG.md` is intended to be consumed by humans, and not machines.
+  - The following sub-sections MAY be used, as appropriate or specified:
+    - `General` - General comments about the release that users should know about.
+    - `Breaking Changes` - Any changes that will break backward compatibility
+      with previous versions. MUST list all breaking changes.
+    - `Bugfixes` - Details of bugs that were fixed. SHOULD list all bug fixes.
+    - `Enhancements` - New features that have been added to the repository. SHOULD
+      list all new features.
   - The file SHOULD contain an `Unreleased` section at the top, which includes
     changes that have not yet been released.
   - The file MUST be in reverse chronological order, with the most recent
     releases at the top of the file, after the `Unreleased` section.
+  - Each release MUST contain a link to the upstream release notes.
+  - Each release SHOULD contain a list of changes from upstream that Splunk has
+    been working on, are relevant to Splunk GDI, or fix outstanding bugs.
+  - Each change coming from upstream MUST bear a label that indicates where the
+    change is coming from. For example: `(Contrib)` or `(Core)`.
```

Comment:
Is the first line

> - Each release MUST contain a link to the upstream release notes.

 not "good enough"?

Do we really need 

> Each release SHOULD contain a list of changes from upstream that Splunk has
    been working on, are relevant to Splunk GDI, or fix outstanding bugs."

?


If we think that it is good enough then I propose the following (OPTION A):

```suggestion
  - Each release MUST contain a link to the upstream release notes.
  - Each release MAY contain a list of changes from upstream that Splunk has
    been working on, are relevant to Splunk GDI, or fix outstanding bugs.
  - Each change coming from upstream MUST bear a label that indicates where the
    change is coming from. For example: `(Contrib)` or `(Core)`.
```

or even simpler (OPTION B)

```suggestion
  - Each release MUST contain a link to the upstream release notes.
```

I would for sure approve OPTION B as part of this PR. I propose to create a separate PR for proposing how to define changes coming from upstream.